apis/server/copy_bridge.go

```diff
@@ -0,0 +1,97 @@
+package server
+
+import (
+	"context"
+	"encoding/base64"
+	"encoding/json"
+	"errors"
+	"io"
+	"net/http"
+
+	"github.com/alibaba/pouch/pkg/httputils"
+
+	"github.com/gorilla/mux"
+)
+
+func (s *Server) putContainersArchive(ctx context.Context, rw http.ResponseWriter, req *http.Request) error {
+	name := mux.Vars(req)["name"]
+	path := req.FormValue("path")
+
+	if name == "" {
+		return httputils.NewHTTPError(errors.New("name can't be empty"), http.StatusBadRequest)
+	}
+	if path == "" {
+		return httputils.NewHTTPError(errors.New("path can't be empty"), http.StatusBadRequest)
+	}
+	noOverwriteDirNonDir := httputils.BoolValue(req, "noOverwriteDirNonDir")
+	copyUIDGID := httputils.BoolValue(req, "copyUIDGID")
+
+	return s.ContainerMgr.ExtractToDir(ctx, name, path, copyUIDGID, noOverwriteDirNonDir, req.Body)
+}
+
+func (s *Server) headContainersArchive(ctx context.Context, rw http.ResponseWriter, req *http.Request) error {
+	name := mux.Vars(req)["name"]
+	path := req.FormValue("path")
+
+	if name == "" {
+		return httputils.NewHTTPError(errors.New("name can't be empty"), http.StatusBadRequest)
+	}
+	if path == "" {
+		return httputils.NewHTTPError(errors.New("path can't be empty"), http.StatusBadRequest)
+	}
+
+	stat, err := s.ContainerMgr.StatPath(ctx, name, path)
+	if err != nil {
+		return err
+	}
+
+	statJSON, err := json.Marshal(stat)
+	if err != nil {
+		return err
+	}
+
+	rw.Header().Set(
+		"X-Pouch-Container-Path-Stat",
```

Comment:
In swagger, you add `X-Docker-Container-Path-Stat`, but here is `X-Pouch-Container-Path-Stat`. Is it right?